package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "sequelize",
-  "description": "Multi dialect ORM for Node.JS",
-  "version": "0.0.0-development",
-  "maintainers": [
-    "Pedro Augusto de Paula Barbosa <papb1996@gmail.com>"
+  "description": "An easy-to-use and promise-based multi SQL dialects ORM tool for Node.js",
+  "version": "6.12.0-beta.1",
```

Comment:
Please revert this line

Reply:
Context: https://github.com/semantic-release/cli#what-it-does (see number 4)

Reply:
Changed ✔️